tests/checkpoint/test_checkpoint.py

```diff
@@ -651,12 +649,8 @@ def test_checkpoint_sorts_actions(self, validation_definition: ValidationDefinit
         Pydantics and mocks.
         Ideally, this would be tested through the public `run()` method.
         """
-        pd_action = PagerdutyAlertAction(
-            name="my_pagerduty_action", api_key="api_key", routing_key="routing_key"
-        )
-        og_action = OpsgenieAlertAction(name="my_opsgenie_action", api_key="api_key")
         data_docs_action = UpdateDataDocsAction(name="my_docs_action")
-        actions: List[CheckpointAction] = [pd_action, og_action, data_docs_action]
+        actions: List[CheckpointAction] = [data_docs_action]
```

Comment:
This test is suppose to verify sorting actions. If there is only 1 action, this test is no longer verifying this. Could you add actions to replace the ones you removed?

Reply:
I'll try! But I'm definitely out of my comfort zone here so if the update I make doesn't do the trick I'd love to pair on this

Reply:
Thanks for doing this! Happy to pair.